dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/condition/HttpUrlCondition.kt

```diff
@@ -9,11 +9,18 @@ import okhttp3.mockwebserver.RecordedRequest
  * Requests without <code>requestUrl</code>s are excluded from matching.
  */
 abstract class HttpUrlCondition : Condition {
-    override fun isRequestMatching(request: RecordedRequest) =
-        when (val url = request.requestUrl) {
+    internal open val httpMethod: HTTPMethod = HTTPMethod.GET
```

Comment:
Forcing the `httpMethod` to `GET` will break the current library functionality that matches any HTTP method request.

Should we allow matching any HTTP method by default to avoid breaking the current functionality?
I think it makes more sense to keep the current functionality.
@koral--

Reply:
Yeah, definitely we should avoid breaking changes.

Reply:
Done 🙂